README.md

```diff
@@ -5,65 +5,61 @@ Stores all the manifests and corresponding image-infos to test OSbuild.
 
 ### Update workflow
 
-The update workflow is made of 3 distinct steps:
+#### 1/ on GitHub: trigger the Gitlab CI
 
-#### First step, triggering the CI
-
-The update workflow starts with an `Update DB` GitHub action that is manually
-triggered. This action imports a subset of the manifests from osbuild-composer,
-specifically those CI is able to build.
-
-Importing manifests which cannot be built by CI will leave the
-DB in a dirty state. For this reason they are excluded.
+The update workflow creates a branch that is pushed to [this GitLab
+repository](https://gitlab.com/redhat/services/products/image-builder/ci/manifest-db).
+Using a GitLab `trigger token` the GitHub action starts the CI with a
+different workflow than the one used to run the regression testing.
 
-Once the import is done, a commit is created and pushed to [this GitLab repository](https://gitlab.com/redhat/services/products/image-builder/ci/manifest-db).
+#### 2/ on CI: Manifest generation
 
-#### Second step, running the CI
+A job synchronise with the latest composer version and run the `gen-manifests`
```

Comment:
This sentence sounds confusing to me. What about "The job downloads the latest osbuild-composer sources from upstream, generates all test manifests using the `gen-manifest` tool and stores them as an artifact."?